paasta_tools/cli/utils.py

```diff
@@ -738,10 +738,7 @@ def short_to_full_git_sha(short, refs):
     :param refs: A list of refs in the git repository
     :return: The full git sha or None if one can't be found
     """
-    for sha in list(set(refs.values())):
-        if sha.startswith(short):
-            return sha
-    return None
+    return [sha for sha in set(refs.values()) if sha.startswith(short)]
```

Comment:
I disagree - this is a library function, and library functions should be un opinionated about how return values should be interpreted. The function is 'just' pattern matching, there is no need to force 0 matches to mean exit(1).  If it shouldn't be in a library, move it back up into the script.